scripts/ci-e2e.sh

```diff
@@ -93,7 +105,7 @@ prerequisites_powervs(){
     # Assigning PowerVS variables
     export IBMPOWERVS_SSHKEY_NAME=${IBMPOWERVS_SSHKEY_NAME:-"powercloud-bot-key"}
     export IBMPOWERVS_IMAGE_NAME=${IBMPOWERVS_IMAGE_NAME:-"capibm-powervs-centos-streams8-1-24-2"}
-    export IBMPOWERVS_SERVICE_INSTANCE_ID=${BOSKOS_RESOURCE_ID:-"0f28d13a-6e33-4d86-b6d7-a9b46ff7659e"}
+    export IBMPOWERVS_SERVICE_INSTANCE_ID=${BOSKOS_RESOURCE_ID:-"d53da3bf-1f4a-42fa-9735-acf16b1a05cd"}
```

Comment:
any specific reason for changing this id? this is stale anyways and hope we are already overriding this while calling this script.

Reply:
Yes, this value will be overridden. The earlier value was pointing to a deleted service instance, hence changing it. It is a placeholder value and wouldn't make much difference.